src/v2/api/controllers/GroupController.ts

```diff
@@ -232,29 +233,33 @@ export class GroupController {
 
   @ApiBearerAuth()
   @ApiOkResponse({
-    type: ShortDisciplinesResponse,
+    type: [ShortDisciplineResponse],
```

Comment:
Ok, you've changed ShortDisciplinesResponse to [ShortDisciplineResponse]. However, ther must be a `ShortDisciplinesResponse`